label_studio/data_import/serializers.py

```diff
@@ -21,7 +21,14 @@ class Meta:
 
 class FileUploadSerializer(serializers.ModelSerializer):
     file = serializers.FileField(use_url=False)
+    size = serializers.SerializerMethodField()
 
     class Meta:
         model = FileUpload
-        fields = ['id', 'file']
+        fields = ['id', 'file', 'size']
+
+    def get_size(self, obj):
```

Comment:
```suggestion
    def get_size(self, obj) -> int | None:
```

This will help Spectacular accurately type the field in responses; otherwise it will just be a string